scripts/test-proxy-startup-with-env-vars.sh

```diff
@@ -2,24 +2,38 @@
 set -e  # Exit on any error
 
 ##### This script is meant to be run in ci #####
-# It tests that the env vars defined in .env.example.holesky are correct.
+# It tests that the env vars defined in the specified environment file are correct.
 # It starts the eigenda-proxy with those env vars, waits 5 seconds, and then kills the proxy.
-# If any deprecated flags are still being used in .env.example.holesky, the script will fail.
+# If any deprecated flags are still being used in the specified environment file, the script will fail.
+
+ENV_FILE=$1
+
+# Check if the environment file exists
+if [ ! -f "$ENV_FILE" ]; then
+    echo "Error: Environment file $ENV_FILE does not exist"
+    echo "Current working directory: $(pwd)"
+    echo "Files in current directory:"
+    ls -la
+
+    exit 1
+fi
+
+echo "Using environment file: $ENV_FILE"
 
 # build the eigenda-proxy binary
 make
 
-# Start the eigenda-proxy with the env vars defined in .env.example.holesky
-set -a; source .env.example.holesky; set +a
+# Start the eigenda-proxy with the env vars defined in the specified environment file
+set -a; source "$ENV_FILE"; set +a
 ./bin/eigenda-proxy &
 PID=$!
 
 # Ensure we kill the process on script exit
 trap "kill $PID" EXIT
 
-# Wait 5 seconds for startup to happen
-echo "sleeping 5 seconds to let the proxy start up"
-sleep 5
+# Wait 75 seconds for startup to happen (actual startup can take up to ~50 seconds, due to SRS parsing)
```

Comment:
wtf...? is this new? what changed?

Reply:
A couple things:
1. g2 srs parsing takes close to 30 seconds, and previously wasn't happening at all (since it wasn't required for v1)
2. with v2 enabled, we are instantiating two backends instead of one, so that adds some time too